src/doc_builder/autodoc.py

```diff
@@ -329,9 +329,10 @@ def get_source_link(obj, page_info, version_tag_suffix="src/"):
     """
     Returns the link to the source code of an object on GitHub.
     """
-    package_name = page_info["package_name"]
+    repo_name = page_info["repo_name"]
```

Comment:
Can we use a get here? not sure the `page_info` will always have everything.
So try `repo_name`, then default to `package_name` if not passed.